app/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,65 @@
+class WebhooksController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: [ :stripe ]
+  skip_authentication
+
```

Comment:
Think we need to skip csrf protection here:

```rb
protect_from_forgery except: :stripe
```

https://docs.stripe.com/webhooks?verify=check-signatures-library#csrf-protection

Reply:
Ahh wait, that's already covered by skipping `:verify_authenticity_token`, hadn't released that could be used as an alternative to `protect_from_forgery`